nesta/core/schemas/tier_1/mappings/datasets/gtr_mapping.json

```diff
@@ -0,0 +1,115 @@
+{
+    "mappings": {
+        "dynamic": "strict",
+        "properties": {
+            "date_end_project": {
+                "type": "date"
+            },
+            "date_start_project": {
+                "type": "date"
+            },
+            "json_funding_project": {
+                "properties": {
+                    "amount": {
+                        "type": "integer"
+                    },
+                    "category": {
+                        "type": "keyword"
+                    },
+                    "currencyCode": {
+                        "type": "keyword"
+                    },
+                    "end": {
+                        "type": "date"
+                    },
+                    "start": {
+                        "type": "date"
+                    }
+                },
+                "type": "nested"
+            },
+            "json_outcomes_project": {
```

Comment:
Not sur about this one: does this mean that [the type is dtermined at runtime? or that it can have multiple types?](https://www.elastic.co/guide/en/elasticsearch/reference/current/dynamic-field-mapping.html)
What's the reason for using this?

Reply:
Sorry I should have put a note here (I will write into the metadata):

The `dynamic`/`nested` bit will mean that this field is basically just an unstructured nosql store, and that the nested types will be determined on ingestion. This will lead to bad behaviour for inconsistently formatted numeric, date and boolean fields. However, I took this decision because there are currently fourteen additional/supplementary `outcomes` tables in GtR, each with subtly different schemas:

```
artisticandcreativeproducts 
collaborations              
disseminations              
furtherfundings             
impactsummaries             
intellectualproperties      
keyfindings                 
policyinfluences            
products                    
publications                
researchdatabaseandmodels   
researchmaterials           
softwareandtechnicalproducts
spinouts    
```                

which all reflect different outcomes after the projects finished. For most projects, I believe that these are empty, but this is in desperate need of EDA before we can decide how to structure the data more formally.

The plan was to dump all of these fields (where they exist) directly into this `json_outcomes_project` field, where they can be analysed later. I'm confident that structure can be added to unify the interface to the `outcomes` fields, but I think it's a little out of scope for this PR

Reply:
OK thanks a lot for the explainer, one this is up we'll discuss about how to make EDA on this kind of fields, thanks!